zthumbor/loaders/zloader.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+from six.moves import urllib
+from tornado.concurrent import return_future
+from thumbor.loaders import LoaderResult, file_loader, http_loader
+from tc_aws.loaders import s3_loader
+from thumbor.context import Context
+from .helpers import (
+    get_url_params, sign_is_valid, THUMBOR_S3_TYPE, THUMBOR_LOCAL_FILE_TYPE,
+    THUMBOR_EXTERNAL_TYPE
+)
+
+from typing import Any, Callable
+
+def get_not_found_result():
+    # type: () -> LoaderResult
+    result = LoaderResult()
+    result.error = LoaderResult.ERROR_NOT_FOUND
+    result.successful = False
+    return result
+
+@return_future  # type: ignore # This was giving mypy error for missing generic datatype?
```

Comment:
He mypy was reporting a missing generic type annotation (which I didn't knew how to fix) so for the timing I added a `type: ignore` but I would really like to know what happened here.
To reproduce just remove the type declaration comment and run mypy.

Reply:
@neiljp may be able to help debug this.

Reply:
The simplest fix for this is annotating with `Callback[..., Any]`, but you may know more about the specific parameter and return types based on the context.